src/server/server_family.cc

```diff
@@ -1281,8 +1281,11 @@ void PrintPrometheusMetrics(uint64_t uptime, const Metrics& m, DflyCmd* dfly_cmd
   const auto& conn_stats = m.facade_stats.conn_stats;
   AppendMetricWithoutLabels("max_clients", "Maximal number of clients", GetFlag(FLAGS_maxclients),
                             MetricType::GAUGE, &resp->body());
-  AppendMetricWithoutLabels("connected_clients", "", conn_stats.num_conns, MetricType::GAUGE,
-                            &resp->body());
+  AppendMetricHeader("connected_clients", "", MetricType::GAUGE, &resp->body());
+  AppendMetricValue("connected_clients", conn_stats.num_conns_main, {"listener"}, {"main"},
+                    &resp->body());
+  AppendMetricValue("connected_clients", conn_stats.num_conns_other, {"listener"}, {"other"},
+                    &resp->body());
```

Comment:
@adiholden Won't this break our dashboards ? Since now we need to use the label as well on our `queries` ? 

Double checking here that this is ok for us.

Reply:
double checked with @Pothulapati we should be ok with the label. It won't break our dashboards.